aries-site/src/pages/components/textinput.js

```diff
@@ -17,20 +26,144 @@ const TextInput = () => (
     />
     <ContentSection>
       <Subsection name={title} level={1} topic={topic}>
+        <SubsectionText>{page.description}</SubsectionText>
+        <Example
+          designer="https://designer.grommet.io/textinput?id=HPE-design-system-taylor-seamans-hpe-com&mode=edit"
+          docs="https://v2.grommet.io/textinput?theme=hpe#props"
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/textinput/TextInputExample.js"
+          figma="https://www.figma.com/file/E40Oo2MpdRNMyVaJqfJ0Wd/HPE-Text-Input-Component"
+        >
+          <TextInputExample />
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          The TextInput component allows the user to input shorter forms of data
+          and content. Passwords and tags can also be used with the TextInput
+          component. Style can be variable, based upon the use case and customer
+          need that will elicit user confidence in success.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="About TextInput" level={3} gap="small">
+        <SubsectionText>
+          Text input fields perform text validation. Some use cases for
```

Comment:
maybe we can mention that having pre filled inputs like password or email to reduce error for users to know what they are putting in.